src/components/look-controls.js

```diff
@@ -42,15 +65,32 @@ module.exports.Component = registerComponent('look-controls', {
     var canvasEl = document.querySelector('a-scene').canvas;
 
     // Mouse Events
-    canvasEl.addEventListener('mousedown', this.onMouseDown.bind(this), true);
-    canvasEl.addEventListener('mousemove', this.onMouseMove.bind(this), true);
-    canvasEl.addEventListener('mouseup', this.releaseMouse.bind(this), true);
-    canvasEl.addEventListener('mouseout', this.releaseMouse.bind(this), true);
+    canvasEl.addEventListener('mousedown', this.onMouseDown, true);
+    canvasEl.addEventListener('mousemove', this.onMouseMove, true);
+    canvasEl.addEventListener('mouseup', this.releaseMouse, true);
+    canvasEl.addEventListener('mouseout', this.releaseMouse, true);
+
+    // Touch events
+    canvasEl.addEventListener('touchstart', this.onTouchStart);
+    canvasEl.addEventListener('touchmove', this.onTouchMove);
+    canvasEl.addEventListener('touchend', this.onTouchEnd);
+  },
+
+  removeEventListeners: function () {
+    var sceneEl = this.el.sceneEl;
+    var canvasEl = canvasEl && canvasEl.canvas;
```

Comment:
```canvasEl``` is going to be always ```undefined``` isn't it? As is not a global variable, you should query the same way you did in ```setupMouseControls```: 
```var canvasEl = document.querySelector('a-scene').canvas;```
That's why right now all the lookcontrol listeners are still active for each camea